include/uvc_ros_driver.h

```diff
@@ -43,293 +43,227 @@
-	uvc_error_t initAndOpenUvc();
-	int setParam(const std::string &name, float val);
-	void sendCameraParam(const int camera_number, const double fx,
-			     const double fy, const Eigen::Vector2d &p0,
-			     const float k1, const float k2, const float r1,
-			     const float r2, const Eigen::Matrix3d &H);
-	void setCalibration(CameraParameters camParams);
-	inline void deinterleave(const uint8_t *mixed, uint8_t *array1,
-				 uint8_t *array2, size_t mixedLength,
-				 size_t imageWidth, size_t imageHeight);
-	inline void selectCameraInfo(int camera, sensor_msgs::CameraInfo **ci);
-
-public:
-	uvcROSDriver(ros::NodeHandle nh)
-		: nh_(nh), node_name_(ros::this_node::getName()) {};
-	~uvcROSDriver();
-	void uvc_cb(uvc_frame_t *frame);
-	/**
-	 * initialize device, set up topic publishers and compute homography for
-	 * the
-	 * cameras
-	 */
-	void initDevice();
-	/**
-	 * setup uvc stream
-	 */
-	void startDevice();
-	// getter and setter for different internal variables
-	bool getUseOfAITMsgs()
-	{
-		return enable_ait_vio_msg_;
-	};
-	void setUseOFAITMsgs(bool enable)
-	{
-		enable_ait_vio_msg_ = enable;
-	};
-	bool getFlip()
-	{
-		return flip_;
-	};
-	void setFlip(bool flip)
-	{
-		flip_ = flip;
-	};
-	bool getUseOfDepthMap()
-	{
-		return depth_map_;
-	};
-	void setUseOfDepthMap(bool depth_map)
-	{
-		depth_map_ = depth_map;
-	};
-	bool getCalibrationParam()
-	{
-		return set_calibration_;
-	};
-	void setCalibrationParam(bool calibration)
-	{
-		set_calibration_ = calibration;
-	};
-	int getNumberOfCameras()
-	{
-		return n_cameras_;
-	};
-	void setNumberOfCameras(int n_cameras)
-	{
-		n_cameras_ = n_cameras;
-
-		switch (n_cameras) {
-		case 10:
-			//camera_config_ = 0x01F;
-			camera_config_ = 0x3FF;
-			break;
-		case 8:
-			//camera_config_ = 0x00F;
-			camera_config_ = 0x1EF;
-			break;
-
-		case 6:
-			camera_config_ = 0xE7;
-			break;
-
-		case 4:
-
-
-			camera_config_ = 0x63;
-			break;
-
-		case 2:
-		default:
-			camera_config_ = 0x21;
-			break;
-		}
-	};
-	int getCameraConfig()
-	{
-		return camera_config_;
-	};
-	//	void setCameraConfig(int camera_config)
-	//	{
-	//		camera_config_ = camera_config;
-	//	};
-	CameraParameters getCameraParams()
-	{
-		return camera_params_;
-	};
-	void setCameraParams(const CameraParameters &camera_params)
-	{
-		camera_params_ = camera_params;
-	};
-	void getHomographyMapping(
-		std::vector<std::pair<int, int>> &homography_mapping)
-	{
-		homography_mapping = homography_mapping_;
-	};
-	void setHomographyMapping(
-		const std::vector<std::pair<int, int>> &homography_mapping)
-	{
-		homography_mapping_ = homography_mapping;
-	};
-	int getCalibrationMode()
-	{
-		return calibration_mode_;
-	};
-	void setCalibrationMode(int calibration_mode)
-	{
-		calibration_mode_ = calibration_mode;
-
-		// update modulo_ variable also
-		if (calibration_mode != 0) {
-			modulo_ = 12;
-		}
-	};
+    // update modulo_ variable also
+    if (calibration_mode != 0) {
+      modulo_ = 4;
```

Comment:
Can we call this in a better way since it's likely we'll have another "module" variable to throttle down (crazy) IMU rate? For instance what about "modulo_downsample_imgs_" and "modulo_downsample_imu_"?